OpenOversight/app/auth/views.py

```diff
@@ -60,7 +60,7 @@ def before_request():
         and not current_user.confirmed
         and request.endpoint
         and request.endpoint[:5] != "auth."
-        and request.endpoint != "static"
+        and request.endpoint not in ["static", "bootstrap.static"]
```

Comment:
Exclude the `bootstrap.static` folder (used by the new bootstrap-flask dependency to serve assets) so stylesheets and scripts load correctly when the user is unconfirmed